web/src/components/sidebar/QuaySidebar.tsx

```diff
@@ -19,24 +22,30 @@ interface SideNavProps {
   component: JSX.Element;
 }
 
-const routes: SideNavProps[] = [
-  {
-    isSideNav: true,
-    navPath: NavigationPath.organizationsList,
-    title: 'Organizations',
-    component: <OrganizationsList />,
-  },
-  {
-    isSideNav: true,
-    navPath: NavigationPath.repositoriesList,
-    title: 'Repositories',
-    component: <RepositoriesList organizationName={null} />,
-  },
-];
-
 export function QuaySidebar() {
   const location = useLocation();
   const sidebarState = useRecoilValue(SidebarState);
+  const quayConfig = useQuayConfig();
+  const routes: SideNavProps[] = [
+    {
+      isSideNav: quayConfig?.config?.BRANDING.quay_io ? true : false,
```

Comment:
In the config file it would just be another field underneath `Branding` - something like
```
BRANDING:
  quay_io: true
```
There isn't any code on the backend that handles this since only the frontend looks for this value and adds the overview to the sidebar if it's true